### PR TITLE
Notebook on web scraping with tool use

### PR DESCRIPTION
**Pull request by ekzhu**
## Why are these changes needed?

Provide an example to scrap web pages.

## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Comment by ekzhu** [comment]
@areibman let me know if you want to make improvement over this.

**Review by areibman — APPROVED** [review]
Looks good to me!

**Review by sonichi — COMMENTED** [review]
Would you like to render this notebook on the website?

**Comment by jackgerrits** [comment]
> Would you like to render this notebook on the website?

I think we should

**Review by jackgerrits — COMMENTED** [review]
In the front matter the tags use spaces instead of `-`. Also `scrapping` -> `scraping`

File: website/docs/Examples.md
Comment (jackgerrits) [inline website/docs/Examples.md:1]: I don't know how useful it is to have this text based example page in addition to the other one. If "categories" is the big missing thing then we can add developing that to the roadmap of the notebooks page

**Review by jtrugman — APPROVED** [review]

File: website/docs/Examples.md
Comment (ekzhu) [reply website/docs/Examples.md:1]: Added to documentation roadmap.

**Pull request merged**